spec/javascripts/highlight_component_spec.js

```diff
@@ -190,6 +196,17 @@ describe("Helpers.documentationUrl", function () {
     )
   });
 
+  it("creates the correct URL for 'gem' components", function () {
+    expect(
+      Helpers.documentationUrl({
+        prefix: "gem-c",
+        name: "label"
+      })
+    ).toEqual(
+      "https://govuk-static.herokuapp.com/component-guide/label"
```

Comment:
This assumes static has the latest gem. I think it'd be better to link to the app’s component guide that's rendering the page.

eg Something like this: https://github.com/alphagov/govuk-toolkit-chrome/blob/8a5b7b6cfce8361059cfa4e7fab28550099aafb3/src/highlight-component.js#L67